zuul-core/src/test/java/com/netflix/zuul/origins/OriginNameTest.java

```diff
@@ -27,13 +26,9 @@
 @RunWith(JUnit4.class)
 public class OriginNameTest {
```

Comment:
This needs more unit tests, at least:
* verification of equality check
* Tests for when niws name and vip are different
*

Reply:
See above PR.